docs/guac-external-api/init.md

```diff
@@ -1,19 +1,35 @@
-# This file documents the requests which guac makes to contacts another guac node to start a channel
-
-(actual endpoint names, ports, types WIP)
+# This file documents the requests which guac makes to contacts another guac node
 
 ## Propose Channel
 Tells a neighbor if you would like to accept a channel from them
 
-Request data type: `Channel` (maybe a different type, but basically all the info you need to call the
+Endpoint: /propose
+
+Request data type: `Channel` (basically all the info you need to call the
```

Comment:
This might be bad naming, but the `external` api docs is designed to document the interaction between guac and another instance of guac, where the `internal` api docs are meant to document the interaction between an instance of an guac_http and another application over http